frontera/contrib/scrapy/converters.py

```diff
@@ -19,24 +19,25 @@ def to_frontier(self, scrapy_request):
             cookies = scrapy_request.cookies
         else:
             cookies = dict(sum([list(d.items()) for d in scrapy_request.cookies], []))
+
+        meta = {}
         cb = scrapy_request.callback
         if callable(cb):
-            cb = _find_method(self.spider, cb)
+            meta[b'scrapy_callback'] = _find_method(self.spider, cb)
+
         eb = scrapy_request.errback
         if callable(eb):
-            eb = _find_method(self.spider, eb)
+            meta[b'scrapy_errback'] = _find_method(self.spider, eb)
 
         scrapy_meta = scrapy_request.meta
-        meta = {}
         if b'frontier_request' in scrapy_meta:
             request = scrapy_meta[b'frontier_request']
             if isinstance(request, FrontierRequest):
                 meta = request.meta
             del scrapy_meta[b'frontier_request']
 
         meta.update({
-            b'scrapy_callback': cb,
-            b'scrapy_errback': eb,
+            b'scrapy_priority': scrapy_request.priority,
```

Comment:
@rolando what do you think? It looks fine. Even better than before, but I would used default priority.